Formula/jabba.rb

```diff
@@ -0,0 +1,32 @@
+class Jabba < Formula
+  desc "(cross-platform) Java Version Manager"
+  homepage "https://github.com/shyiko/jabba"
+  url "https://github.com/shyiko/jabba/archive/0.9.1.tar.gz"
+  sha256 "5e00ceec91afdea5ef2f2b5ffc764e552bb9837c1d69d7e1375136da46b15abd"
+  head "https://github.com/shyiko/jabba.git"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    dir = buildpath/"src/github.com/shyiko/jabba"
+    dir.install buildpath.children
+    cd dir do
+      ldflags = "-X main.version=#{version}"
+      system "glide", "install"
+      system "go", "build", "-ldflags", ldflags, "-o", "bin/jabba"
```

Comment:
Sorry, slight misunderstanding here.

```ruby
system "go", "build", "-ldflags", ldflags, "-o", bin/"jabba"
```
And then remove the:
```ruby
bin.install "bin/jabba"
```
line.

Reply:
I see, thanks. Done.